.eslintrc.json

```diff
@@ -10,14 +10,9 @@
         "Atomics": "readonly",
         "SharedArrayBuffer": "readonly"
     },
-    "parser": "@typescript-eslint/parser",
     "parserOptions": {
-        "ecmaVersion": 2018,
-        "sourceType": "module"
+        "ecmaVersion": 2018
     },
-    "plugins": [
-        "@typescript-eslint"
-    ],
     "rules": {
     }
```

Comment:
No change should be made to eslintrc file ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)
[see correct file](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)